src/main/com/intellij/lang/jsgraphql/ide/editor/GraphQLIntrospectionHelper.java

```diff
@@ -365,8 +365,10 @@ void createOrUpdateIntrospectionOutputFile(String schemaText, IntrospectionOutpu
 
     void setEditorTextAndFormatLines(String text, FileEditor fileEditor) {
         if (fileEditor instanceof TextEditor) {
+            // IntelliJ only allows the \n linebreak inside editor documents.
+            String normalizedText = text.replaceAll("\\r\\n?", "\n");
```

Comment:
Discussion:

Should this be moved into the `printIntrospectionJsonAsGraphQL` method, so that it can be tested with the existing `testPrintIntrospectionJsonAsGraphQL`? Or should this strictly be treated as a UI driven rule.

Reply:
I think it's good to keep it here for now